frontend/README.md

```diff
@@ -0,0 +1,52 @@
+# TTADP Frontend
+
+A React frontend initialized with Create React App
+
+## Getting Started
+
+There are yarn commands for starting/testing/linting the frontend defined a directory up. `yarn start` or `yarn docker:start` a directory up will start both the backend and frontend. `yarn docker:test` or `yarn test:all` will test the frontend and the backend. However if you want to just run the frontend tests/linter you can run yarn commands in this directory. Install dependencies with `yarn install`. `yarn start` will fire up the development server. Use `yarn test` and `yarn lint` for running tests and the linter.
+
+### Yarn Commands
+
+| Yarn Command | Description |
+|-|-|
+| `yarn start` | Start the frontend development server |
+| `yarn build` | Build a production bundle |
+| `yarn test` | Start the test watcher |
+| `yarn test:ci` | Run unit tests a single time outputting results to xml files |
+| `yarn lint` | Run the linter |
+| `yarn lint:ci` | Run the linter outputting results to xml files |
+
+## Create React App
```

Comment:
Do we need all this (through the end of the page)? If it's helpful that's cool.  I'm wondering if we're giving free advertising to facebook. 😄

Reply:
:laughing: at the very least I think we should keep the link to create-react-app so if someone needs to mess with the build system they know where to look.